scratch-gui/src/lib/pyright-client.js

```
@@ -0,0 +1,213 @@
+import {
+    CompletionRequest,
+    CompletionResolveRequest,
+    DiagnosticTag,
+    HoverRequest,
+    InitializeRequest,
+    NotificationType,
+    RequestType,
+    createMessageConnection,
+} from "vscode-languageserver-protocol";
+
+import { STUB } from "./python-worker";
```

ESLint

[error] There should be no space after '{'.

(object-curly-spacing)